sql/mob_spawn_points.sql

```diff
@@ -60632,7 +60636,7 @@ INSERT INTO `mob_spawn_points` VALUES (16838975,'Ypotrill','Ypotryll',471,63,22,
 INSERT INTO `mob_spawn_points` VALUES (16838976,'Ypotrill','Ypotryll',471,90,24,250,218);
 INSERT INTO `mob_spawn_points` VALUES (16838977,'Ypotrill','Ypotryll',471,122,24,303,170);
 INSERT INTO `mob_spawn_points` VALUES (16838978,'Ephemeral_clionid','Ephemeral Clionid',430,127,23,295,102);
-INSERT INTO `mob_spawn_points` VALUES (16838979,'Alkonost','Alkonost',413,0,0,0,0);
+INSERT INTO `mob_spawn_points` VALUES (16838979,'Alkonost','Alkonost',90200,52,30,413,88);
```

Comment:
Use whatever ID dsp has there, or if dsp had zero, make a new one in their file that is +1 to whatever last one in their file is.